CONTRIBUTING.rst

```diff
@@ -113,11 +96,11 @@ Install local dependencies:
 Run the project automated tests
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-With a running database:
+Within docker container:
 
 .. code-block:: console
 
-    (venv) $ pytest  # Test the code with the current interpreter
+    $ docker-compose exec procrastinate pytest  # Test the code with the latest python
 
 Or
```

Comment:
Before, with this `Or`, we were comparing the execution of the tests to the execution of all the checks. Now this is confusing because we use docker-compose for the execution of pytest, but not for the execution of tox.